crypto/fipsmodule/aes/asm/aesni-x86.pl

```diff
@@ -694,10 +696,9 @@ sub aesni_generate6
 	&and	("ebp",1<<28|1<<11);	# AVX and XOP bits
 	&cmp	($rounds,256);
 	&je	(&label("14rounds"));
-	&cmp	($rounds,192);
-	&je	(&label("12rounds"));
 	&cmp	($rounds,128);
 	&jne	(&label("bad_keybits"));
+	# 192-bit key support was removed
```

Comment:
This comment should should appear two lines above where the 192-bit code chunk was removed.